File: src/main/java/com/procurement/notice/model/entity/ReleaseEntity.java
package com.procurement.notice.model.entity;

import java.util.Date;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class ReleaseEntity {

    private String cpId;

    private String ocId;

    private Date releaseDate;

    private String releaseId;

    private String stage;

    private String jsonData;
}

File: src/main/kotlin/com/procurement/notice/model/entity/ReleaseEntity.kt
package com.procurement.notice.model.entity

import com.fasterxml.jackson.annotation.JsonInclude
import java.util.*

data class ReleaseEntity(

    var cpId: String,

    var ocId: String,

    var publishDate: Date,

    var releaseDate: Date,

    var releaseId: String,

    var stage: String,

    var jsonData: String,

    @field:JsonInclude(JsonInclude.Include.NON_NULL)
    var status: String? = ""
)
